### ISSUE TITLE
Seems that bid/ask prices are swapped in wex exchange

### ISSUE THREAD

**Issue by rubobaquero** [ccxt-dev/ccxt#389]
It seems that bid and ask prices are swapped in wex->fetch_ticker() response. For example:

```
$wex = new \ccxt\wex();
$wex->load_markets();
print_r($wex->fetch_ticker("ETH/EUR"));
```

Returns a bid price (272.384) higher than the ask price (270.70566). 

```
Array
(
    [symbol] => ETH/EUR
    [timestamp] => 1508856580000
    [datetime] => 2017-10-24T14:49:40.000+00:00
    [high] => 278.387
    [low] => 246.90808
    [bid] => 272.384
    [ask] => 270.70566
    [vwap] => 
    [open] => 
    [close] => 
    [first] => 
    [last] => 271.08476
    [change] => 
    [percentage] => 
    [average] => 262.64754
    [baseVolume] => 1707.00159
    [quoteVolume] => 443983.21075
    [info] => Array
        (
            [high] => 278.387
            [low] => 246.90808
            [avg] => 262.64754
            [vol] => 443983.21075
            [vol_cur] => 1707.00159
            [last] => 271.08476
            [buy] => 272.384
            [sell] => 270.70566
            [updated] => 1508856580
        )

)
```

**Comment by kroitor** [comment]
Hi! Good catch! Thx for your report! Will fix shortly and get back to you!

**Comment by kroitor** [comment]
This issue is fixed as of 1.9.243+. WEX bid/asks are now set properly. Thx again! Let us know if you have any other questions or difficulties. Cheers!
